anki/tags.py

```diff
@@ -139,13 +139,16 @@ def addToStr(self, addtags, tags):
         return self.join(self.canonify(currentTags))
 
     def remFromStr(self, deltags, tags):
-        "Delete tags if they don't exists."
+        "Delete tags if they exist."
+        def wildcard(pat, str):
+            return '*' in pat and re.search(
+                pat.replace('*', '.*'), str, re.IGNORECASE)
         currentTags = self.split(tags)
         for tag in self.split(deltags):
             # find tags, ignoring case
             remove = []
             for tx in currentTags:
-                if tag.lower() == tx.lower():
+                if (tag.lower() == tx.lower()) or wildcard(tag, tx):
```

Comment:
Parens not needed (but ok). "wildcard" would be clearer as "is_wildcard" (and any function that is a predicate, in general)